editor.planx.uk/src/@planx/components/Feedback/Public/Public.tsx

```diff
@@ -21,14 +26,39 @@ import { Feedback, FormProps } from "../model";
 import { StyledToggleButtonGroup } from "../styled";
 
 const FeedbackComponent = (props: PublicProps<Feedback>): FCReturn => {
+  const handleSubmitFeedback = async (
+    values: FormProps,
+    { resetForm }: FormikHelpers<FormProps>,
+  ) => {
+    const metadata = await getInternalFeedbackMetadata();
+    const feedback = {
+      userComment: values.feedback,
+      feedbackScore: values.feedbackScore,
+    };
+    const data = {
+      ...metadata,
+      ...feedback,
+      feedbackType: "component" as FeedbackView,
+    };
+    const submitFeedbackResult = await insertFeedbackMutation(data).catch(
+      (err) => {
+        console.error(err);
+      },
+    );
+    props.handleSubmit?.(makeData(props, values, "_feedback"));
+    if (!submitFeedbackResult) {
+      return;
+    }
+
+    resetForm({ values });
```

Comment:
I don't think we need to worry about resetting the form - calling `props.handleSubmit()` will trigger the next card to be rendered when a breadcrumb is added.